src/functions-and-arrays.js

```diff
@@ -29,8 +89,37 @@ const wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(wordsUnique){
+  if(wordsUnique.length === 0){
+    return null
+  } else {
+    let uniqueArray = [];
+    wordsUnique.forEach((wordUnique) => {
+      if(uniqueArray.indexOf(wordUnique) === -1){
+        uniqueArray.push(wordUnique);
+      }
+    });
+    return uniqueArray;
+  }
+}
+
+
```

Comment:
Nice! Could also add the index of the array to the final if statement within the forEach for clarity: 

if (uniqueArray.indexOf(wordsUnique[i]) === -1) {
      uniqueArray.push(wordsUnique[i]);